README.md

```diff
@@ -62,6 +62,13 @@ To install the latest version, run the following:
 pip install tensorflow-addons
 ```
 
+You will also need TensorFlow 2.0 or higher. To use addons:
+
+```python
+import tensorflow as tf
+import tensorflow_addons
```

Comment:
In the examples and README we use `import tensorflow_addons as tfa` Are you okay with adding that alias? Similar to tensorflow_probability as tfp

Reply:
`tfa` works, as well! 🙂 Will make the mod now.